src/Extension/Resources/config/task/RunInstallTask.yaml

```diff
@@ -1,8 +1,8 @@
 ---
-id: "project:install:run"
+id: "install:project:run"
 short_description: "Runs Sprykers install command."
 help: ~
 stage: build
-version: 0.3.1
+version: 0.3.2
```

Comment:
an id has been changed. That's a minor change for beta version.

Reply:
changed. but it doesn't make sense. we don't have a minor or major here. The important thing that the version is up